README.md

```diff
@@ -1,6 +1,6 @@
 # Jira extension for Sourcegraph
 
-A [Sourcegraph extension](https://docs.sourcegraph.com/extensions) that enhances GitHub with better [Atlassian Jira](https://www.atlassian.com/software/jira) integration.
+A [Sourcegraph extension](https://docs.sourcegraph.com/extensions) that enhances GitHub with better [Jira](https://www.atlassian.com/software/jira) integration.
```

Comment:
@emchap You're right, only cloud supports API tokens (server uses [oauth](https://developer.atlassian.com/server/jira/platform/oauth/) or [usr+pass](https://developer.atlassian.com/server/jira/platform/basic-authentication/) it looks like). Since they're sunsetting Jira server over the [next 1-3 years](https://www.atlassian.com/blog/announcements/journey-to-cloud) I think this isn't something we need to support, but I agree with adding a note that it only works for cloud. 
```suggestion
A [Sourcegraph extension](https://docs.sourcegraph.com/extensions) that enhances GitHub with better [Jira Cloud](https://www.atlassian.com/software/jira) integration.
```